src/Mvc/Mvc.Core/src/FromBodyAttribute.cs

```diff
@@ -9,6 +9,9 @@ namespace Microsoft.AspNetCore.Mvc;
 /// <summary>
 /// Specifies that a parameter or property should be bound using the request body.
 /// </summary>
+/// <remarks>
+/// Be default ASP.NET Core runtime delegates the responsibility of reading the body to an input formatter.
```

Comment:
This is true for MVC but not for minimal API, which also uses the same attribute.

Reply:
How does it behave for minipal API then?

Reply:
I think it just reads the form itself directly before passing control to the user's endpoint method using https://learn.microsoft.com/dotnet/api/microsoft.aspnetcore.http.features.iformfeature.readformasync (I haven't dug through the code to check, but that's what I see to remember from doing work on the binding "glue" stuff).

Reply:
Found it:
[Here](https://github.com/dotnet/aspnetcore/blob/98ee50279acfacff2b4ae4b6863956f3c65f34b2/src/Http/Http.Extensions/src/RequestDelegateFactory.cs#L735) and then [here](https://github.com/dotnet/aspnetcore/blob/98ee50279acfacff2b4ae4b6863956f3c65f34b2/src/Http/Http.Extensions/src/RequestDelegateFactory.cs#L1390).

Ok, I need to change the wording. The fun thing that MVC's model binding involves validation, and even System.Text.Json does not care about `[Required]` attrbiute I get 400 Bad Request while for minimal API I get 200.

Reply:
@martincostello , I changed the wording.